src/bin/sccache-dist/token_check.rs

```diff
@@ -152,19 +151,18 @@ impl MozillaCheck {
             .get(url.clone())
             .set_header(header)
             .send()
-            .context("Failed to make request to mozilla userinfo".to_owned())?;
```

Comment:
I think this is fallout from some `sd` foo applied after refactoring the errors :+1: